core/pkg/telemetry/metrics.go

```diff
@@ -69,19 +69,9 @@ func getDurationView(svcName, viewName string, bucket []float64) metric.View {
 	)
 }
 
-func NewOTelRecorder(exporter metric.Reader, serviceName string) *MetricsRecorder {
-	const requestDurationName = "http_request_duration_seconds"
-	const responseSizeName = "http_response_size_bytes"
+func NewOTelRecorder(serviceName string) *MetricsRecorder {
+	meter := global.Meter(serviceName)
```

Comment:
Thanks, yes that's a fair argument. 

The advantage of global provider is that we can use the provider through `global`, similar to the usage here in any location where we need metrics. However, yes drawbacks are there.

Instead of the global approach, I will try to unify startup parameter handling and yield a reader instead. That way, individual metric handlers can build their provider based on the reader.

Reply:
Something worth mentioning: since we are the Application code, we should set the global providers. This way, libraries that have OTel API support can expose traces and metrics :)

Reply:
Updated documentation to have current telemetry overview [1]

[1] - https://github.com/open-feature/flagd/blob/301e74f1571b67a6dd7cc87ef8b86025d698314e/docs/configuration/flagd_telemetry.md